include/BDSFieldEMAcceleration.hh

```diff
@@ -0,0 +1,28 @@
+//
+// Created by Boogert Stewart on 16/08/2024.
```

Comment:
Licence text should be here and also doxygen compatible authorship like the others in the codebase.